### PR TITLE
bugfix: use system time  and current folder to jstack

### PR DESCRIPTION
**Pull request by leizhiyuan**
### Ⅰ. Describe what this PR did

use System.currentTimeMillis() as id, and jstack dump to current folder.

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/seata/seata/issues/3922

### CODE REVIEW

**Comment by slievrly** [comment]
@leizhiyuan add change log https://github.com/seata/seata/blob/develop/changes/en-us/1.5.0.md   https://github.com/seata/seata/blob/develop/changes/1.5.0.md

**Comment by leizhiyuan** [comment]
> @leizhiyuan add change log https://github.com/seata/seata/blob/develop/changes/en-us/1.5.0.md https://github.com/seata/seata/blob/develop/changes/1.5.0.md

done

File: core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java [hunk 1]
```diff
@@ -289,9 +288,9 @@ protected void processMessage(ChannelHandlerContext ctx, RpcMessage rpcMessage)
                         if (allowDumpStack) {
                             String name = ManagementFactory.getRuntimeMXBean().getName();
                             String pid = name.split("@")[0];
-                            int idx = new Random().nextInt(100);
+                            long idx = System.currentTimeMillis();
                             try {
-                                Runtime.getRuntime().exec("jstack " + pid + " >d:/" + idx + ".log");
+                                Runtime.getRuntime().exec("jstack " + pid + " > " + idx + ".log");
```
Comment (jsbxyyx) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java:293]: may be no permission, use current execute user home dir?

Comment (jsbxyyx) [reply core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java:293]: and log.info the command?

**Comment by jsbxyyx** [comment]
rename pr title

File: core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java (hunk 1 above)
Comment (leizhiyuan) [reply core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java:293]: > may be no permission, use current execute user home dir?

I think this operation will  dump the file to the folder of the running path?

Comment (jsbxyyx) [reply core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java:293]: recommended to add logging record commands. view the stack information path from the log information.

Comment (jsbxyyx) [reply core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java:293]: recommended to add logging record commands. view the stack information path from the log information.

Comment (leizhiyuan) [reply core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java]: done

**Review by slievrly — COMMENTED** [review]
@leizhiyuan a small problem about code style, please fix it.

image

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**